hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -52,6 +52,8 @@
   public static final String DISK_CHECK_TIMEOUT_KEY =
       "hdds.datanode.disk.check.timeout";
 
+  static final boolean CHUNK_DATA_VALIDATION_CHECK_DEFAULT = true;
```

Comment:
The value is set to true by default, lets have this as false by default but lets enable this in MiniOzoneCluster